build/charts/theia/values.yaml

```diff
@@ -151,6 +151,7 @@ grafana:
   installPlugins:  
     - https://downloads.antrea.io/artifacts/grafana-custom-plugins/theia-grafana-sankey-plugin-1.0.2.zip;theia-grafana-sankey-plugin
     - https://downloads.antrea.io/artifacts/grafana-custom-plugins/theia-grafana-chord-plugin-1.0.1.zip;theia-grafana-chord-plugin
+    - https://file.io/HWfwKx57UxHq;theia-grafana-dependency-plugin
```

Comment:
You will always need to run `make` under `build/charts` to update the README file for Helm chart when you update the `value.yaml` file.

Reply:
+1 please also take care of the go test failures https://github.com/antrea-io/theia/actions/runs/3589954800/jobs/6042870930 @Dhruv-J